File: JAVA_POO/aula07/Aula07.java
package aula07;

public class Aula07 {

    public static void main(String[] args) {

        Lutador l[] = new Lutador [6];

        l[0] = new Lutador("Pretty Boy", "frances", 31, 1.75f,68.9f,11,1,2);
        l[1] = new Lutador("Putscript", "brasileiro", 29, 1.68f,57.8f,14,3,2);
        l[2] = new Lutador("Snapshadow", "EUA", 35, 1.65f,80.9f,12,1,2);
        l[3] = new Lutador("Dead Code", "australiano", 28, 1.93f,81.6f,13,2,0);
        l[4] = new Lutador("UFOCobol", "brasileiro", 37, 1.70f,119.3f,5,3,4);
        l[5] = new Lutador("Nerdaart", "EUA", 30, 1.81f,105.7f,12,4,2);

        l[0].apresentar();
        l[1].apresentar();
        l[2].apresentar();
        l[3].apresentar();
        l[4].apresentar();
        l[5].apresentar();
    }
}

File: PHP_POO/Aula 07/Aula 07.php
<!doctype html>
<html lang="pt-br">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Aula 07</title>
</head>
<body>
    <?php
        require_once 'Lutador.php';

        $l = array();
        $l = array();
        $l[0] = new Lutador('Vitor Belfort', 'brasileiro', 35, 1.86, 80, 11, 2, 5);
        $l[1] = new Lutador('Putscript', 'francês', 29, 1.68, 63, 14, 2, 3);
        $l[2] = new Lutador('Snapshadow', 'EUA', 20, 1.65, 80.9, 13, 5, 1);
        $l[3] = new Lutador('Anderson Selva', 'brasileiro', 40, 1.80, 73, 28, 2, 2);
        $l[4] = new Lutador('Maguila', 'brasileiro', 60, 1.95, 100, 23, 15, 13);
        $l[5] = new Lutador('Lomaschenko', 'ucraniano', 25, 1.75, 65, 20, 0, 1);

        for($k = 0; $k <5; $k++){
        $l[$k]->apresentar();
        }
    ?>
</body>
</html>